MiniGamesBox Classic/src/main/java/plugily/projects/minigamesbox/classic/utils/version/VersionUtils.java

```diff
@@ -171,20 +172,19 @@ public static SkullMeta setPlayerHead(Player player, SkullMeta meta) {
 
   private static JavaPlugin plugin;
 
-  public static void teleport(Entity entity, Location location) {
+  public static CompletableFuture<Boolean> teleport(Entity entity, Location location) {
```

Comment:
basically the question comes up if we benefit of that in the future as only paper will be "helped" with it and doens't make a real benefit for the overall plugin performance?

Reply:
The performance is unaffected here, paper's teleportAsync method tells in its name that it will loads the chunk (if unloaded) asynchronously to teleport to the specified location without force loading the chunk which blocks the main thread.